cmd/goal/node.go

```diff
@@ -119,18 +122,55 @@ var nodeCmd = &cobra.Command{
 	},
 }
 
+func getMissingCatchpointLabel(URL string) (label string, err error) {
+	resp, err := http.Get(URL)
+	if err != nil {
+		return
+	}
+	if resp.StatusCode != 200 {
+		err = errors.New(resp.Status)
+		return
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return
+	}
+	label = string(body)
+	label = strings.TrimSuffix(label, "\n")
+
+	// check if label is a valid catchpoint label
+	_, _, err = ledgercore.ParseCatchpointLabel(label)
+	if err != nil {
+		return
+	}
+	return
+}
+
 var catchupCmd = &cobra.Command{
```

Comment:
You created a new command but did not add it to the root `nodeCmd`...was that on purpose?